### ISSUE TITLE
Mask dislocation when test

### ISSUE THREAD

**Issue by oblivious137** [DeepSceneSeg/EfficientPS#41]
line 365, mmdet/models/efficientps/efficientPS.py
Directly resize the semantic logits to ref_size may cause dislocation.
It should be resized to pad_shape and cropped to ref_size.
This bug occurs when I use another dataset. I'm not sure whether it occurs in the original dataset.
